packages/gatsby-cli/src/reporter/reporters/index.js

```diff
@@ -1,7 +1,12 @@
-const semver = require(`semver`)
 const { isCI } = require(`ci-info`)
 
-if (semver.satisfies(process.version, `>=8`) && !isCI) {
```

Comment:
Can we still keep node version check - if for some reason `ink` is actually installed in node 6 env, we should not try to use it as it will crash

Reply:
Got it, thanks! Updating now.